v-next/hardhat/test/hre/index.ts

```diff
@@ -3,7 +3,9 @@ import { describe, it } from "node:test";
 
 import { createHardhatRuntimeEnvironment } from "../../src/hre.js";
 import { builtinPlugins } from "../../src/internal/builtin-plugins/index.js";
+import { resolveConfigPath } from "../../src/internal/helpers/config-loading.js";
```

Comment:
I think the `cacelledByParent` errors are a failure in the `before` or `after`.

(I'm commenting in the closest line that github allowed me to, they are above)